notebooks/official/model_monitoring/model_monitoring.ipynb

```diff
@@ -201,11 +201,162 @@
     },
```

Comment:
<p>remove execution output throughout the notebook</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/964/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
Removed and pushed changes.